jeremy-aaron-tre/scripts/article.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+let articles = [];
+
+// COMMENT: What is the purpose of the following function? Why is its name capitalized? Explain the context of "this" within the function. What does "rawDataObj" represent?
+// It's an object constructor which is why it is capitalized. It is used to create instances of an object. This in this context refers to the object that owns the method. 'rawDataObj' is a parameter for the object constructor.
+
+function Article (rawDataObj) {
+  // DONE: Use the JS object that is passed in to complete this constructor function:
+  // Save ALL the properties of `rawDataObj` into `this`
+  this.rawDataObj = rawDataObj;
+}
```

Comment:
the individual properties should be listed out to be more clear, clean, and explicit vs taking in the entire object.